Bottom: f876a45f283ff11c613aa49504a82472ba352f98
Top:    0454a9cf9f04da045152500cc6cd1711efef5208
Author: Parth Shah <parth@linux.ibm.com>
Date:   2019-07-09 11:26:31 +0530

Refresh of sched-topology-introduce

---

diff --git a/include/linux/sched/topology.h b/include/linux/sched/topology.h
index cf041c6ec4ba..cc3c4b621cab 100644
--- a/include/linux/sched/topology.h
+++ b/include/linux/sched/topology.h
@@ -163,6 +163,13 @@ static inline struct cpumask *sched_domain_span(struct sched_domain *sd)
 	return to_cpumask(sd->span);
 }
 
+#ifdef CONFIG_SCHED_SMT
+static inliner struct cpumask *sched_domain_cores(struct sched_domain *sd)
+{
+	return to_cpumask(sd->shared->core_mask);
+}
+#endif
+
 extern void partition_sched_domains(int ndoms_new, cpumask_var_t doms_new[],
 				    struct sched_domain_attr *dattr_new);
